Advanced-Recruit-Training/src/BatteryStateOfChargeService.cpp

```diff
@@ -8,6 +8,11 @@ namespace
 BatteryStateOfChargeService::BatteryStateOfChargeService(double initialStateOfChargePercent)
 : initialStateOfChargePercent_(initialStateOfChargePercent)
 {
+    current_ = 0;
+    ampHoursUsed_ = BATTERY_AMP_HOUR_CAPACITY - BATTERY_AMP_HOUR_CAPACITY*(initialStateOfChargePercent_/100);
```

Comment:
while this appears to be logically correct, perhaps there is a more straightforward way to accomplish this calculation, say without reusing the same variable twice?